padinfo/core/button_info.py

```diff
@@ -12,12 +14,33 @@
 Base:    {}
 Subattr: {}
 Total:   {}
+
+--------------
+{}{}
 """
 
+TEAM_BUTTON_FORMAT = "As {} base ({}x {}): Contributes {}"
+CARD_BUTTON_FORMAT = "As {} base ({}x): Deals {}"
+
+MonsterStat = namedtuple('MonsterStat', 'name id mult att')
+
+team_buttons = [
```

Comment:
sorry i didnt catch this before: these should be ALL_CAPS cos theyre global constants